### PR TITLE
docs: Fix broken yaml indentation

### PR DESCRIPTION
**Pull request by ujihisa**
Commit Message: Fix broken yaml indentation
Additional Description: N/A
Risk Level: Low
Testing: Compare this <https://github.com/ujihisa/envoy/blob/c687265807990f61572a769ccb9776c0d34d97d3/docs/root/configuration/upstream/cluster_manager/cluster_circuit_breakers.rst> with this <https://www.envoyproxy.io/docs/envoy/latest/configuration/upstream/cluster_manager/cluster_circuit_breakers>
Docs Changes: Fix broken yaml indentation
Release Notes: N/A

### CODE REVIEW

**Comment by alyssawilk** [comment]
Thanks for the fix!  Mind sorting out the DCO failure?

**Comment by ujihisa** [comment]
> Thanks for the fix! Mind sorting out the DCO failure?

Oops sorry I missed that. Now it's done!

File: docs/root/configuration/upstream/cluster_manager/cluster_circuit_breakers.rst [hunk 1]
```diff
@@ -11,14 +11,14 @@ The following is an example circuit breaker configuration:
...
-          value: 25.0
-        min_retry_concurrency: 10
+    thresholds:
+      - priority: "DEFAULT"
+        max_requests: 75
+        max_pending_requests: 35
+        retry_budget:
+          budget_percent:
+            value: 25.0
+          min_retry_concurrency: 10
```
Comment (dio) [inline docs/root/configuration/upstream/cluster_manager/cluster_circuit_breakers.rst:21]: Super nit, sorry, for consistency, could you help to follow the indentation for "list item" like e.g. this one: https://www.envoyproxy.io/docs/envoy/v1.15.0/intro/arch_overview/security/ssl#example-configuration? While it is not written anywhere, I tend to see this is how we write yaml in envoy docs.

```yaml
    thresholds:
    - priority: "DEFAULT"
      max_requests: 75
      max_pending_requests: 35
      retry_budget:
        budget_percent:
          value: 25.0
        min_retry_concurrency: 10
```

Thanks, @ujihisa!

Comment (ujihisa) [reply docs/root/configuration/upstream/cluster_manager/cluster_circuit_breakers.rst]: Fixed! :+1:

**Pull request merged**